features/advance_search.feature

```diff
@@ -32,3 +32,7 @@ Feature: Flaw advance search testing
     Scenario: Search flaws with multiple conditions in query filter
       Given I go to the advanced search page
       Then I am able to search flaws with multiple conditions in query filter
+
+    Scenario: Search flaws with osidb related models
+      Given I go to the advanced search page
+      Then I am able to search flaws with osidb related models in query filter
```

Comment:
```
  Scenario: Search flaws with osidb related models                           # features/advance_search.feature:36
    Given I am an analyst AND I am logged into OSIM                          # features/steps/common_steps.py:14 7.851s
    * I set the bugzilla api key and jira api key                            # features/steps/common_steps.py:19 7.460s
    Given I go to the advanced search page                                   # features/steps/common_steps.py:41 0.118s
    Then I am able to search flaws with osidb related models in query filter # features/steps/advance_search.py:189 62.772s
      Traceback (most recent call last):
        File "/Users/axuan/PycharmProjects/osim/venv/lib/python3.9/site-packages/seleniumpagefactory/Pagefactory.py", line 69, in __getattr__
          element = WebDriverWait(self.driver, self.timeout).until(
        File "/Users/axuan/PycharmProjects/osim/venv/lib/python3.9/site-packages/selenium/webdriver/support/wait.py", line 105, in until
          raise TimeoutException(message, screen, stacktrace)
      selenium.common.exceptions.TimeoutException: Message: 
      
      
      During handling of the above exception, another exception occurred:
      
      Traceback (most recent call last):
        File "/Users/axuan/PycharmProjects/osim/venv/lib/python3.9/site-packages/behave/model.py", line 1329, in run
          match.run(runner.context)
        File "/Users/axuan/PycharmProjects/osim/venv/lib/python3.9/site-packages/behave/matchers.py", line 98, in run
          self.func(context, *args, **kwargs)
        File "features/steps/advance_search.py", line 193, in step_impl
          search_data = flaw_page.get_valid_search_keywords_from_created_flaw()
        File "/Users/axuan/PycharmProjects/osim/features/pages/flaw_detail_page.py", line 820, in get_valid_search_keywords_from_created_flaw
          public_flaw_result["mitigation"] = self.get_document_text_field("mitigation")
        File "/Users/axuan/PycharmProjects/osim/features/pages/flaw_detail_page.py", line 305, in get_document_text_field
          field_element = getattr(self, field + 'Text')
        File "/Users/axuan/PycharmProjects/osim/venv/lib/python3.9/site-packages/seleniumpagefactory/Pagefactory.py", line 73, in __getattr__
          raise ElementNotVisibleException(
      seleniumpagefactory.Pagefactory.ElementNotVisibleException: An exception of type TimeoutException occurred. With Element -: mitigationText - locator: (xpath, (//span[contains(text(),'Mitigation')])[last()])



Failing scenarios:
  features/advance_search.feature:36  Search flaws with osidb related models

0 features passed, 1 failed, 0 skipped
0 scenarios passed, 1 failed, 6 skipped
3 steps passed, 1 failed, 29 skipped, 0 undefined
Took 1m18.200s

```

Due to flaw history remove to the bottom of the page, some element's xpath expression need to change.